lib/css/modules/_showImages.scss

```diff
@@ -43,39 +43,42 @@ img {
 
 	}
 
-	&.image,
-	&.video-muted {
-		vertical-align: top !important;
-		float: left;
-		width: 23px;
-		height: 23px;
-		max-width: 23px;
-		max-height: 23px;
-		display: inline-block;
-		background-image: url('../images/expandoButtons.png');
-		margin-right: 6px;
-		padding: 0;
+	&.res-expando-button {
+		&.video.res-media-muted,
+		&.image,
+		&.gallery {
+			vertical-align: top !important;
+			float: left;
+			width: 23px;
+			height: 23px;
+			max-width: 23px;
+			max-height: 23px;
+			display: inline-block;
+			background-image: url('../images/expandoButtons.png');
+			margin-right: 6px;
+			padding: 0;
+		}
 
 		&.commentImg {
 			float: none;
 			margin-left: 4px;
 		}
-	}
 
-	@function _($i) { @return $i * -24px; }
-
-	&.image.collapsedExpando { background-position: 0 _(0); }
-	&.image.collapsedExpando:hover { background-position: 0 _(1); }
-	&.image.expanded { background-position: 0 _(2); }
-	&.image.expanded:hover { background-position: 0 _(3); }
-	&.image.gallery.collapsedExpando { background-position: 0 _(4); }
-	&.image.gallery.collapsedExpando:hover { background-position: 0 _(5); }
-	&.image.gallery.expanded { background-position: 0 _(6); }
-	&.image.gallery.expanded:hover { background-position: 0 _(7); }
-	&.video-muted.collapsed { background-position: 0 _(8); }
-	&.video-muted.collapsed:hover { background-position: 0 _(9); }
-	&.video-muted.expanded { background-position: 0 _(10); }
-	&.video-muted.expanded:hover { background-position: 0 _(11); }
+		@function _($i) { @return $i * -24px; }
+
+		&.image.collapsedExpando { background-position: 0 _(0); }
+		&.image.collapsedExpando:hover { background-position: 0 _(1); }
+		&.image.expanded { background-position: 0 _(2); }
+		&.image.expanded:hover { background-position: 0 _(3); }
+		&.gallery.collapsedExpando { background-position: 0 _(4); }
+		&.gallery.collapsedExpando:hover { background-position: 0 _(5); }
+		&.gallery.expanded { background-position: 0 _(6); }
+		&.gallery.expanded:hover { background-position: 0 _(7); }
+		&.video-muted.collapsed { background-position: 0 _(8); }
```

Comment:
If `.video.res-media-muted` is used above then it should be used here too, or vice versa.